### PR TITLE
arm64: change some format

### PR DESCRIPTION
**Pull request by hujun260**
## Summary
arm64: change some format

fix the comment in https://github.com/apache/nuttx/pull/14980

## Impact
arm64

## Testing
ci ostest

### CODE REVIEW

**Comment by nuttxpr** [comment]
[**\[Experimental Bot, please feedback here\]**](https://github.com/search?q=repo%3Aapache%2Fnuttx+13552&type=issues)



No, this PR does not fully meet the NuttX requirements. While it provides some information, it lacks crucial details.

Here's a breakdown of what's missing:

* **Summary:**
    *  Lacks a clear explanation of *why* the format change is necessary.  Is it for readability? Consistency? Correctness?  Just saying "fix the comment" isn't sufficient.  What was wrong with the comment?
    *  Needs to specify *which* functional part of the code is being changed.  "arm64" is too broad. Is it in a specific driver?  A core architecture file?
    *  Needs to explain *how* the change works. "change some format" is vague.  Provide specifics about the formatting changes.

* **Impact:**
    *  Simply stating "arm64" doesn't explain the impact.  The Impact section requires YES/NO answers for each category.  Even if the answer is NO, it should still explicitly state NO.  If the answer is YES, a description is required. For example:
        * Impact on user: NO
        * Impact on build: NO
        * Impact on hardware: YES (affects arm64 architecture, potentially impacting performance or code size - needs details)
        * ...etc...

* **Testing:**
    *  "ci ostest" is insufficient. While CI testing is important, it should be supplemented with specific local testing details.  What build host was used? What specific arm64 target board and configuration?  
    *  The requirement specifically asks for "Testing logs before change" and "Testing logs after change." These are missing entirely.  Simply saying the change "works as intended" is not verifiable. Show evidence!  Even if the change is purely cosmetic, provide *some* form of output demonstrating the change.


In short, the PR needs to be much more descriptive and provide concrete evidence of testing and impact assessment.  Each section of the NuttX requirements needs to be addressed thoroughly.

File: arch/arm64/src/common/arm64_syscall.c [hunk 1]
```diff
@@ -193,8 +195,11 @@ uint64_t *arm64_syscall(uint64_t *regs)
 
         restore_critical_section(tcb, cpu);
         break;
-      case SYS_switch_context:
 
+      case SYS_switch_context:
+#ifdef CONFIG_ARCH_ADDRENV
```
Comment (xiaoxiang781216) [inline arch/arm64/src/common/arm64_syscall.c:200]: move after line 211

File: arch/arm64/src/common/arm64_syscall.c [hunk 2]
```diff
@@ -193,8 +195,11 @@ uint64_t *arm64_syscall(uint64_t *regs)
...
+        addrenv_switch(NULL);
```
Comment (xiaoxiang781216) [inline arch/arm64/src/common/arm64_syscall.c:201]:
```suggestion
        addrenv_switch(tcb);
```

File: arch/arm64/src/common/arm64_syscall.c [hunk 3]
```diff
@@ -184,7 +184,9 @@ uint64_t *arm64_syscall(uint64_t *regs)
   switch (cmd)
     {
       case SYS_restore_context:
-
+#ifdef CONFIG_ARCH_ADDRENV
+        addrenv_switch(NULL);
```
Comment (xiaoxiang781216) [inline arch/arm64/src/common/arm64_syscall.c:188]:
```suggestion
        addrenv_switch(tcb);
```

File: arch/arm64/src/common/arm64_syscall.c [hunk 4]
```diff
@@ -184,7 +184,9 @@ uint64_t *arm64_syscall(uint64_t *regs)
   switch (cmd)
     {
       case SYS_restore_context:
-
+#ifdef CONFIG_ARCH_ADDRENV
```
Comment (xiaoxiang781216) [inline arch/arm64/src/common/arm64_syscall.c:187]: move after line 196

Comment (hujun260) [reply arch/arm64/src/common/arm64_syscall.c]: ok

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**